include/sundials/priv/sundials_cli.h

```diff
@@ -0,0 +1,148 @@
+/* -----------------------------------------------------------------
```

Comment:
Can this be moved to `src/sundials/sundials_cli.h` to limit scope? I don't think any of the declarations are used in `include`.

Reply:
`include/sundials/priv` is installed because some header files include function implementations so the "private" error handling functions and macros need to be installed so they're accessible in these functions. If these implementations were made internal then `include/sundials/priv` could be removed.